profiles/common/modules/features/nexteuropa_feature_set/nexteuropa_feature_set.features.user_permission.inc

```diff
@@ -0,0 +1,384 @@
+<?php
+/**
+ * @file
+ * nexteuropa_feature_set.features.user_permission.inc
+ */
+
+/**
+ * Implements hook_user_default_permissions().
+ */
+function nexteuropa_feature_set_user_default_permissions() {
```

Comment:
Should we provide this permission implementation in the scope of the NEPT-182?
This looks more like a task for a separate ticket. Correct me if I'm wrong.